### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by RitaDee**
### In this PR, I implemented the following:

- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Add your own styles to improve the look and feel.
 - Use React basic styling (inline styles, importing CSS and pre-processed CSS files).
- For every page, you should create a [Route](url).
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/styles/Home.module.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+}
```
Comment (VICTOR-CHUKWU) [inline src/styles/Home.module.css:28]: - [ ] nice animation on page render :+1:

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
-    <div className="App">
-      <Calculator />
-    </div>
+    <Routes>
+      <Route path="/" element={<Layout />}>
+        <Route path="/calculator" element={<Calculator />} />
+        <Route path="/quote" element={<Quote />} />
+        <Route path="/" element={<Home />} />
+      </Route>
+    </Routes>
```
Comment (VICTOR-CHUKWU) [inline src/App.js:17]: - [ ] Good use of react router :clap:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @RitaDee , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] good use of react router

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (RitaDee) [reply src/App.js:17]: Thank you @VICTOR-CHUKWU.

**Pull request merged**